SSH section update (#30)
csc-training/geocomputing_course: materials/where_to_go.md

## Connecting to the supercomputer via SSH

> We have accessed the supercomputer via the webinterface in order to not overwhelm you with setups before the course. However, this way may not always be the most convenient. Direct SSH connection likely works better.

Reviewer: ```suggestion
We have accessed the supercomputer via the webinterface in order to not overwhelm you with setups before the course. However, this way may not always be the most convenient. You can also directly connect your computer to the supercomputer via SSH.
```

:::{admonition} Connecting with SSH clients
:class: seealso, dropdown

- SSH clients give command-line access to a supercomputer or any other Linux server.

> - Basic SSH connection will not allow displaying remote graphics, but it is possible to set up with extra settings, but in case of Puhti web interface is recommended.

Reviewer: ```suggestion
- Basic SSH connection will not allow displaying remote graphics, but it is possible to set up with extra settings. 
```

Author: Why not web interface?

- SSH clients:
   - Mac and Linux have Terminal for SSH connections.
   - In Windows:
      - `Command Prompt` or `Powershell` are always avaialbe and can be used for basic connections.
      - Special tools like [PuTTY](https://www.putty.org/) or [MobaXterm](https://mobaxterm.mobatek.net/) provide more options, inc possibility to save settings, but need installation.
- [CSC Docs: Connecting to CSC supercomputers](https://docs.csc.fi/computing/connecting/)

> - [LUMI Docs: Get started](https://docs.lumi-supercomputer.eu/firststeps/).

Reviewer: If you want to avoid typing your password every time you log in to the supercomputer, you can also set up ssh-keys, which in addition also make your connection more secure. You can find instructions for different tools in [CSC Docs: Setting up ssh-keys](https://docs.csc.fi/computing/connecting/#setting-up-ssh-keys).
